CRM/Utils/System/Drupal9.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC https://civicrm.org/licensing
+ */
+
+/**
+ * Drupal specific stuff goes here.
+ */
+class CRM_Utils_System_Drupal9 extends CRM_Utils_System_Drupal8 {}
```

Comment:
@eileenmcnaughton I genuinely don't see a need for this at all given that we are not changing the UF between Drupal 8 -> Drupal 9

Reply:
@seamuslee001 yeah - it feels 'right' to me to put this here because it seems inevitable there will be some variance at some point - but TBH my involvement in this one is just that I was trying to resolve an old stalled PR. (I've already tried to close it twice :-))

Reply:
I would merge it if this was not included

Reply:
OK - I've removed it - I guess putting this to bed is more important than my feeling it should be added